### PR TITLE
feat: prepare repo for C1 dev

### PR DESCRIPTION
**Pull request by shadinaif**
feat: prepare repo for C1 dev

See notion doc for details https://www.notion.so/correlation-one/Running-an-extra-ora2-grading-MFE-f04843851ed94da38a6c8906e7da676e

### CODE REVIEW

**Comment by berithpy** [comment]
LGTM

**Review by berithpy — APPROVED** [review]

File: .env.development [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+LOGIN_URL='http://lmsbe.local.correlation-one.com:8020/login'
+LOGOUT_URL='http://lmsbe.local.correlation-one.com:8020/logout'
 LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
 LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
 LOGO_WHITE_URL=https://edx-cdn.org/v3/default/logo-white.svg
 LOGO_POWERED_BY_OPEN_EDX_URL_SVG=https://edx-cdn.org/v3/stage/open-edx-tag.svg
 FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
 CSRF_TOKEN_API_PATH='/csrf/api/v1/token'
-REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
+REFRESH_ACCESS_TOKEN_ENDPOINT='http://lmsbe.local.correlation-one.com:8020/login_refresh'
```
Comment (andres0sorio) [inline .env.development:13]: Interesting, was the 18000 port wrong?

Comment (andres0sorio) [reply .env.development:13]: I see ... everything move to port 8020

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
     "lint-fix": "fedx-scripts eslint --fix --ext .jsx,.js src/",
     "semantic-release": "semantic-release",
     "start": "fedx-scripts webpack-dev-server --progress",
+    "start-dev": "fedx-scripts webpack-dev-server --progress --host lmsbe.local.correlation-one.com",
```
Comment (andres0sorio) [inline package.json:16]: Does `--host` accept the argument without the "http://" ?

**Review by andres0sorio — APPROVED** [review]
Changes look straight forward.

File: package.json (hunk 2 above)
Comment (shadinaif) [reply package.json:16]: yes it does 👍🏼

**Pull request merged**